update_version.sh

```diff
@@ -1,12 +1,26 @@
 #!/usr/bin/env bash
-version_in_branch=$(cat VERSION)
-echo "MR branch:" $version_in_branch
 
-git fetch -q origin master
+if [ -f  pyproject.toml ]; then
+  version_file="pyproject.toml"
```

Comment:
What do you think if we will make it rather configurable than have it hardcoded? And based on argument value will decide which command to execute? 

In that case (even tho it is not probably possible) we will avoid problems with overriding command in case both files (VERSION and pyproject.toml) will be presented it project